21.3-1/21.3-1-g6803368d-0ubuntu1~series.3/oci_bionic_integration_tests.txt

```diff
@@ -0,0 +1,5984 @@
+2021-09-22 23:29:45 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c hostname
+2021-09-22 23:29:45 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c cloud-id
+2021-09-22 23:29:46 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'cat /run/cloud-init/result.json'
+2021-09-22 23:29:46 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'cat /etc/netplan/50-cloud-init.yaml'
+2021-09-22 23:29:46 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c systemd-analyze
+2021-09-22 23:29:46 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'systemd-analyze blame'
+2021-09-22 23:29:47 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'cloud-init analyze show'
+2021-09-22 23:29:48 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'cloud-init analyze blame'
+2021-09-22 23:29:49 INFO      pycloudlib.instance:instance.py:176 executing: sudo -- sh -c 'cat /var/log/cloud-init.log'
+
+=== `systemd-analyze` before:
+Startup finished in 5.750s (kernel) + 30.009s (userspace) = 35.759s
+graphical.target reached after 27.282s in userspace
+=== `systemd-analyze` after:
+Startup finished in 5.847s (kernel) + 18.500s (userspace) = 24.348s
+graphical.target reached after 14.898s in userspace
+
+=== `systemd-analyze blame` before (first 10 lines):
+         14.695s snapd.seeded.service
+          7.158s cloud-init.service
+          1.620s cloud-config.service
+          1.438s dev-sda1.device
+          1.266s pollinate.service
+          1.213s apparmor.service
+          1.167s cloud-init-local.service
+          1.106s cloud-final.service
+          1.013s snapd.service
+           590ms networkd-dispatcher.service
+=== `systemd-analyze blame` after (first 10 lines):
+          6.556s cloud-init.service
+          3.149s snapd.service
+          2.201s dev-sda1.device
+          2.112s cloud-config.service
+          1.531s cloud-final.service
+          1.345s cloud-init-local.service
+           982ms networkd-dispatcher.service
+           817ms lxd-containers.service
+           602ms dev-loop0.device
+           594ms dev-loop2.device
+
+=== `cloud-init analyze show` before:')
+Finished stage: (init-local) 00.10800 seconds
+Finished stage: (init-network) 06.47500 seconds
+Finished stage: (modules-config) 00.62000 seconds
+Finished stage: (modules-final) 00.35200 seconds
+Total Time: 7.55500 seconds
+=== `cloud-init analyze show` after:')
+Finished stage: (init-local) 00.10900 seconds
+Finished stage: (init-network) 05.87900 seconds
+Finished stage: (modules-config) 00.73200 seconds
+Finished stage: (modules-final) 00.56100 seconds
+Total Time: 7.28100 seconds
+Finished stage: (init-network) 00.48400 seconds
+Total Time: 0.48400 seconds
+
+=== `cloud-init analyze blame` before (first 10 lines): ')
+-- Boot Record 01 --
+     05.14200s (init-network/search-OpenStack)
```

Comment:
This image looks to be coming up as OpenStack still, I'm thinking we should **NOT** opinionate images with a /etc/cloud/cloud.cfg.d/99_oracle.cfg:
```yaml
datasource_list: [Oracle]
```.

We need to take an action item once we release to get CPC team to add that config setting to Oracle bionic cloud images.